src/contacts/ConsoleReader.java

```diff
@@ -0,0 +1,59 @@
+package contacts;
+
+import contacts.service.Validator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.regex.Pattern;
+
+public class ConsoleReader {
+    private static BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
+    public static String getStringFromConsole(String message) throws IOException {
```

Comment:
please do not throw any exception from this class

Reply:
Done